### DISCUSSION TITLE
does this work with big sur 11.2?

### DISCUSSION THREAD

**Discussion by snowFal** [chris1111/Wireless-USB-Big-Sur-Adapter#69] [Q&A]
just like the title says

**Comment by chris1111** [comment]
> just like the title says

Yes
Works also with Big Sur 11.3 Beta

**Reply by snowFal** [reply]
thank you for your reply! 
also another question if you don’t mind.

my laptop’s wifi doesn’t turn on it just won’t work as i spilled some water on it, my question is will the adapter work if i installed ur link using an ethernet cable?

**Reply by chris1111** [reply]
> thank you for your reply!
> also another question if you don’t mind.
> 
> my laptop’s wifi doesn’t turn on it just won’t work as i spilled some water on it, my question is will the adapter work if i installed ur link using an ethernet cable?

No its for USB adapter wifi not for a cable

**Reply by snowFal** [reply]
no, what i meant is that i wouldn’t be able to install your link because i don’t have wifi so what if i use an ethernet cable first to install it and then use the adapter after i manage to install the link

**Reply by snowFal** [reply]
hi me again, this worked great for me but i can’t see my 5G network i can only see the 2.4G one why is that? also one more question, if i update my mac again will the USB continue to work? 

btw i’m using the TL-WN823N

**Reply by chris1111** [reply]
> hi me again, this worked great for me but i can’t see my 5G network i can only see the 2.4G one why is that? also one more question, if i update my mac again will the USB continue to work?
> 
> btw i’m using the TL-WN823N

5G Realtek adapter never work on macOS
